landcover.mss

```diff
@@ -356,11 +357,17 @@
   [feature = 'power_generator'][zoom >= 10],
   [feature = 'power_sub_station'][zoom >= 13],
   [feature = 'power_substation'][zoom >= 13] {
+    polygon-fill: @industrial;
+    [zoom >= 15] {
       polygon-fill: @power;
-      [zoom >= 12] {
-        line-width: 0.4;
-        line-color: darken(@power, 40%);
```

Comment:
I also thought about defining `@power`, `@power-line` and `@high-zoom-power` - as separate values, without depending on `@industrial`.

In the end I used this version as it seemed to be easier to customize.